easybuild/easyconfigs/n/n2v/n2v-0.3.2-foss-2022a-CUDA-11.7.0.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonBundle'
+
+name = 'n2v'
+version = '0.3.2'
+versionsuffix = '-CUDA-11.7.0'
```

Comment:
I'm curious, why did you remove `CUDA` as a direct dependency (which then prevented you from using the `%(cudaver)s` template here), is it the case that `n2v` itself doesn't use `CUDA`, only via `CSBDeep`, and you wanted to make that clear? 

If that's the case, is this a distinction we should be looking out for in PRs?